src/main/groovy/au/org/ala/merit/command/SaveReportDataCommand.groovy

```diff
@@ -121,6 +122,10 @@ class SaveReportDataCommand implements Validateable {
             // Setting the activityId in the payload is necessary for ecodata to update the activity as well as outputs.
             activity.activityId = activityId
 
+            if (report.organisationId) {
+                activity.organisationId = report.organisationId
+            }
+
             result = activityService.update(activityId, activity)
```

Comment:
This is a good catch - assuming that existing org reports don't have this change applied do we need to write a script to add the organisationId to existing activities?  Probably the routine that actually creates these activities with the reports should be adding the organisation id?

Reply:
@chrisala just created a PR for the script